app/src/main/res/values/strings.xml

```diff
@@ -106,4 +106,7 @@
     <string name="title_completed">Completed</string>
     <string name="cancel">Cancel</string>
     <string name="update">Update</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.hello_blank_fragment` appears to be unused
 Unused resources make applications larger and slow down builds.